plugin/pkg/scheduler/factory/plugins.go

```diff
@@ -61,6 +61,11 @@ const (
 	DefaultProvider = "DefaultProvider"
 )
 
+var (
+	AlgorithmProviders = ""
+	providerCounter    = 1
```

Comment:
Hi @mikedanese and @lavalamp 

Thanks for all the advice. I have learned much:) And I need a little more time to fix all the changes you have proposed.